test/saithrift/tests/sail3.py

```diff
@@ -1477,3 +1477,192 @@ def runTest(self):
+               '00:11:31:33:44:59',
+               '00:11:32:33:44:60',
+               '00:11:33:33:44:61',
+               '00:11:34:33:44:62',
+               '00:11:35:33:44:63',
+               '00:11:36:33:44:64',
+               '00:11:37:33:44:65',
+               '00:11:38:33:44:66']
+    
+    
+    def setup_rifs(self, first_rif_port, port_list):
+        for i in range(self.first_changing_port,first_rif_port):
+            self.lag_members.append(sai_thrift_create_lag_member(self.client, self.lag, port_list[i]))
+        for i in range(first_rif_port,self.total_changing_ports):
+            self.port_rifs.append(sai_thrift_create_router_interface(self.client, self.vr_id, 1, port_list[i], 0, self.v4_enabled, self.v6_enabled, ''))
+        for i in range(self.total_changing_ports - first_rif_port):
+            sai_thrift_create_neighbor(self.client, self.addr_family, self.port_rifs[i], "10.10.%s.1" % str(i+1), self.mac_pool[i])
+            self.nhops.append(sai_thrift_create_nhop(self.client, self.addr_family, "10.10.%s.1" % str(i+1), self.port_rifs[i]))
+        self.nhops.append(sai_thrift_create_nhop(self.client, self.addr_family, "10.10.16.1" , self.lag_rif))
+        self.nhop_group = sai_thrift_create_next_hop_group(self.client, self.nhops)
+        sai_thrift_create_route(self.client, self.vr_id, self.addr_family, "10.10.0.0", self.ip_mask, self.nhop_group)    
+
+
+    def teardown_rifs(self, first_rif_port, port_list):
+        sai_thrift_remove_route(self.client, self.vr_id, self.addr_family, "10.10.0.0", self.ip_mask, self.nhop_group)    
+        self.client.sai_thrift_remove_next_hop_from_group(self.nhop_group, self.nhops)
+        self.client.sai_thrift_remove_next_hop_group(self.nhop_group)
+        for nhop in self.nhops:
+            self.client.sai_thrift_remove_next_hop(nhop)
+        del self.nhops[:]
+        for i in range(self.total_changing_ports - first_rif_port):
+            sai_thrift_remove_neighbor(self.client, self.addr_family, self.port_rifs[i], "10.10.%s.1" % str(i+1), self.mac_pool[i])
+        print self.port_rifs
+        for rif in self.port_rifs:
+            self.client.sai_thrift_remove_router_interface(rif)
+        del self.port_rifs[:]
+        for lag_member in self.lag_members:
+            self.client.sai_thrift_remove_lag_member(lag_member)
+        del self.lag_members[:]
+    
+
+    def polarizationCheck(self,packets,avg):
+        if (avg < 150):
+            self.assertTrue((packets >= (avg * 0.65)),"Not all paths are equally balanced, %s" % packets)
+            self.assertTrue((packets <= (avg * 1.35)),"Not all paths are equally balanced, %s" % packets)
+        else:
+            self.assertTrue((packets >= (avg * 0.8)),"Not all paths are equally balanced, %s" % packets)
+            self.assertTrue((packets <= (avg * 1.2)),"Not all paths are equally balanced, %s" % packets)
+        
+    def send_and_verify_packets(self, first_rif_port, port_list):
```

Comment:
change the parameter to send_and_verify_packets(self, lag_size, total_port_count)